.github/workflows/python.yml

```diff
@@ -0,0 +1,65 @@
+name: Python
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  linux:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        python-version: [ "3.8", "3.9", "3.10" ]
+    defaults:
+      run:
+        working-directory: python
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        lfs: true
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+        cache-dependency-path: 'python/setup.py'
+    - name: Build manylinux wheel
+      run: ./tools/build_wheel.sh $(echo cp${{ matrix.python-version }} | sed "s/\.//")
+    - name: Pip install
+      run: |
+        python -m pip install $(ls wheels/*.whl)[test]
+    - name: Run python tests
+      run: |
+        pytest -x -v --durations=10
+  mac:
+    runs-on: macos-12
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.10"
+    - name: Install dependencies
+      run: |
+        brew update
+        brew install apache-arrow protobuf
```

Comment:
don't need to install parquet separately?

Reply:
and also arrow-python arrow-dataset?

Reply:
on mac, `apache-arrow` installs all of them, this is a mega-package.